Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -1,15 +1,16 @@
 variables:
-    SolutionBaseName: 'Dfc.App.JobCategories'
-    BuildPlatform: 'any cpu'
-    BuildConfiguration: 'release'
+  ArmTemplateRoot: '$(Build.SourcesDirectory)\Resources\ArmTemplates'     
+  SolutionBaseName: 'Dfc.App.JobCategories'
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'release'
 
 resources:
   repositories:
   - repository: self
   - repository: dfc-devops
     type: github
     name: SkillsFundingAgency/dfc-devops
-    ref: refs/tags/v1.8.5
+    ref: refs/tags/v1.10.3
```

Comment:
Note that if you plan to use the dfc-armtemplate-deploy task group in the release, this needs to be v1.8.6

Reply:
I did have the tag at v1.8.6 - this copies the Resources folder into the build outputs. Discussing on Slack if fixing the dfc-armtemplate-deploy task group (bump to v2) is a better approach.

Reply:
There is now a dfc-armtemplate-deploy version 2.* (preview) which can be used to deploy the artifact from a 1.10.* build